lib/BloqadeQMC/src/ising/Rydberg.jl

```diff
@@ -69,86 +69,37 @@ function make_prob_vector(H::Type{<:AbstractRydberg}, V::UpperTriangular{T}, Ω:
     return ops, p, energy_shift
 end
 
+total_hx(H::Rydberg)::Float64 = sum(H.Ω) / 2
```

Comment:
can we have a more readable word on `hx`?

Reply:
We'll check later whether this function gets used at all :)